base/sparse/sparsematrix.jl

```diff
@@ -318,7 +318,218 @@ function sparse_IJ_sorted!{Ti<:Integer}(I::AbstractVector{Ti}, J::AbstractVector
     return SparseMatrixCSC(m, n, colptr, I, V)
 end
 
-## sparse() can take its inputs in unsorted order (the parent method is now in csparse.jl)
+"""
+    sparse(I, J, V, [m, n, combine])
+
+Create a sparse matrix `S` of dimensions `m` x `n` such that `S[I[k], J[k]] = V[k]`. The
+  `combine` function is used to combine duplicates. If `m` and `n` are not specified, they
+  are set to `maximum(I)` and `maximum(J)` respectively. If the `combine` function is not
+  supplied, duplicates are added by default. All elements of `I` must satisfy
+  `1 <= I[k] <= m`, and all elements of `J` must satisfy `1 <= J[k] <= n`.
+
+For additional documentation and an expert driver, see `Base.SparseArrays.sparse!`.
+"""
+function sparse{Tv,Ti<:Integer}(
```

Comment:
Placing arguments all on the first line makes the `function sparse{...}(...` and ` function sparse!{...}(... ` lines exceed the suggested line-length limit. For `sparse` the line overrun isn't too bad, so I placed the arguments on the first line as suggested. But doing the same for `sparse!` yields an exceedingly long line that is somewhat difficulty to parse, so for now I left that line as is --- broken into multiple lines illustrating the arguments' logical grouping. What is the right thing to do here? Exceed the line-length limit grossly, or break from the arguments-on-first-line style? Thanks!